### DISCUSSION TITLE
Strange behavior after inserting waypoints

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5558] [General]
Hi dev team,

on my leg from GCTS to EDDS I had strange experiences while programing the route. After insertation of the RWY 25 and SID VAST8F the destination changed to a waypoint and wasn´t EDDS anymore. It was the waypoint DITION. So any unknown bug (might be MS or Asobo), a few waypoints were added after EDDS.

Nevertheless I did another INIT via simbrief and added the waypoints of the SID VAST8F manually. These waypoints are MERAN/LPC/LARYS/SARAY.

As you can see on the picture, the active waypoint is MERAN but not shown on the ND .... 

![grafik](https://user-images.githubusercontent.com/76531893/128470201-19e38929-6fb8-47f3-9efe-b9d56cb181db.png)

greets Klaus

**Comment by tracernz** [comment]
All work on flight planning, guidance, and the ND is happening on the custom FPM branch at the moment, where these systems are totally replaced.
